km_api/know_me/tests/models/test_emergency_contact_model.py

```diff
@@ -0,0 +1,46 @@
+from know_me import models
+from django.core.exceptions import ValidationError
+import pytest
+
+
+def test_create(km_user_factory):
+    """
+    Test creating an emergency contact.
+    """
+    models.EmergencyContact.objects.create(
+        km_user=km_user_factory(),
+        name='First Last',
+        relation='Parent',
+        phone_number='19199199199',
+        alt_phone_number='12345678909',
+        email='hi@gmail.com')
+
+
+def test_invalid_phone_number(km_user_factory):
+    """
+    Test creating an emergency contact with invalid phone number
+    """
+    with pytest.raises(ValidationError):
+        ec = models.EmergencyContact.objects.create(
+            km_user=km_user_factory(),
+            name='First Last',
+            relation='Parent',
+            phone_number='asdfasdfghjdjhfjasdfghjkjhgfdfghdf',
+            alt_phone_number='12312341234',
+            email='hi@gmail.com')
```

Comment:
If you wanted to save some typing you could use the factory and pass in an invalid `phone_number`. This would also allow for addition/deletion of fields to the contact model in the future without this test breaking.
```python
with pytest.raises(ValidationError):
    contact = emergency_contact_factory(phone_number='foo')
    contact.full_clean()
```